tools/data_gen/src/main.rs

```diff
@@ -1,9 +1,15 @@
 use argh::FromArgs;
 use calyx::{errors::CalyxResult, frontend, ir};
+use rand::Rng;
 use serde_json::{json, Map, Value};
 use std::collections::HashMap;
 use std::path::{Path, PathBuf};
 
+// *How to use*
+// run: cargo run -p data_gen -- <calyx file> to generate data w/ all 0s and
+// type int
+// add -f true if you want random values of type fix<32,16>
+
```

Comment:
We should add some documentation on this so that people know it exists. Either add it in this PR or open an issue about it